### DISCUSSION TITLE
Slow export times?

### DISCUSSION THREAD

**Discussion by micksjunior** [mifi/lossless-cut#1140] [Q&A]
Anyone else getting super slow export times all of a sudden? Its taking me like 10 minutes now for a 10 gb video versus a few minutes before

**Comment by mifi** [comment]
is it exactly the same video as before? did anything else change? same version of losslesscut? which settings?

**Reply by micksjunior** [reply]
It is not the same exact video, but it is the same type of long game footage. Just a few days ago everything was running fine on all types of drives but it is even noticeably slow on everything including an internal SSD. I updated to newest version with no improvement yesterday and my settings have always been on default.

**Comment by mifi** [comment]
would be nice if you could share a JSON report from the help menu
